src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/CustomDescription.tsx

```diff
@@ -45,42 +48,55 @@ const TxInfo = styled.div`
   padding: 8px 8px 8px 16px;
 `
 
-const MultiSendCustomData = ({ tx, order }: { tx: MultiSendDetails; order: number }): React.ReactElement => {
+const TxInfoDetails = ({ data }: { data: DataDecoded }): React.ReactElement => (
+  <TxInfo>
+    <TxDetailsMethodName size="lg">
+      <strong>{data.method}</strong>
```

Comment:
nitpick: here we're using `strong` tag but everywhere else in this file component `<Bold />` is used, was it intended?

Reply:
Bold is a whole component just to put a `<b/>` tag, don't think it is necessary. Also, I prefer the semantic nature of `strong`.

That being said, I just removed this strong, in favor of the `strong` prop in the `TxDetailsMethodName` component :grimacing:

Reply:
I don't think that strong's semantics have to do anything with what we want to there: https://www.w3schools.com/tags/tag_strong.asp

Reply:
Looking at your commit, it seems that you didn't understand my point. My point is that there are 4 occurrencies of <Bold /> component in this file and it would be great if we keep the things consistent

Reply:
the issue for tracking: #1215